features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mssql.sql

```diff
@@ -1,9 +1,16 @@
 CREATE TABLE DM_DEVICE_TYPE (
-     ID INT IDENTITY NOT NULL,
+     ID INTEGER IDENTITY NOT NULL,
      NAME VARCHAR(300) DEFAULT NULL,
      PRIMARY KEY (ID)
 );
 
+CREATE TABLE DM_DEVICE_CERTIFICATE (
+      ID                    INTEGER IDENTITY NOT NULL,
```

Comment:
Fix formatting.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/commit/99ff4cbc10dbc82d5dd7dfed72ce6f1bb11cf3e1